src/main/java/com/laytonsmith/abstraction/bukkit/BukkitMCAnvilInventory.java

```diff
@@ -0,0 +1,80 @@
+package com.laytonsmith.abstraction.bukkit;
+
+import com.laytonsmith.abstraction.MCAnvilInventory;
+import com.laytonsmith.abstraction.MCItemStack;
+import org.bukkit.inventory.AnvilInventory;
+
+public class BukkitMCAnvilInventory extends BukkitMCInventory implements MCAnvilInventory {
+
+	AnvilInventory ai;
+
+	public BukkitMCAnvilInventory(AnvilInventory inventory) {
+		super(inventory);
+		ai = inventory;
+	}
+
+	@Override
+	public MCItemStack getFirstItem() {
+		return new BukkitMCItemStack(ai.getItem(0));
+	}
+
+	@Override
+	public MCItemStack getSecondItem() {
+		return new BukkitMCItemStack(ai.getItem(1));
+	}
+
+	@Override
+	public MCItemStack getResult() {
+		return new BukkitMCItemStack(ai.getItem(2));
+	}
+
+	@Override
+	public int getMaximumRepairCost() {
+		return ai.getMaximumRepairCost();
+	}
+
+	@Override
+	public int getRepairCost() {
+		return ai.getRepairCost();
+	}
+
+	@Override
+	public int getRepairCostAmount() {
+		return ai.getRepairCost();
```

Comment:
I believe that this should call `getRepairCostAmount()` instead.

Reply:
Indeed it should! Updated.